src/controller/blogsController.ts

```diff
@@ -1,67 +1,64 @@
 import { Request, Response } from "express";
-import {sql} from '@vercel/postgres'
+import { sql } from "@vercel/postgres";
 
 class BlogController {
-    async getAllBlogs(req: Request, res: Response) {
-        try {
-            const blogs = await sql `SELECT * FROM blogs`;
-            return res.status(200).json(blogs)
-        
-        } catch (error) {
-              console.error(error)
-              return res.status(500).send('Internal Server Error')  
-        }     
+  async getAllBlogs(req: Request, res: Response) {
+    try {
+      const blogs = await sql`SELECT * FROM blogs`;
+      return res.status(200).json(blogs);
+    } catch (error) {
+      console.error(error);
+      return res.status(500).send("Internal Server Error");
     }
+  }
 
-    async getBlogById(req: Request, res: Response) {
-        const {id} = req.params
+  async getBlogById(req: Request, res: Response) {
+    const { id } = req.params;
 
-        try {
-            const blog = await sql `SELECT * FROM blogs WHERE id = ${id}`
-            return res.status(200).json(blog)
-        } catch (error) {
-            console.error(error)
-            return res.status(404).send('Blog not found')
-        }
+    try {
+      const blog = await sql`SELECT * FROM blogs WHERE id = ${id}`;
+      return res.status(200).json(blog);
+    } catch (error) {
+      console.error(error);
+      return res.status(404).send("Blog not found");
     }
+  }
 
+  async createBlog(req: Request, res: Response) {
+    const { title, discription, content, author } = req.body;
 
-    async createBlog(req: Request, res: Response) {
-        const {title, discription, content, author } = req.body;
-
-        try {
-            const newBlog = await sql`INSERT INTO blogs (title, description, content, author) VALUES (${title}, ${discription}, ${content}, ${author}) RETURNING *`;
-            return res.status(201).json(newBlog);
-          } catch (error) {
-            console.error(error);
-            return res.status(500).send('Internal Server Error');
-          }
+    try {
+      await sql`INSERT INTO blogs (title, description, content, author) VALUES (${title}, ${discription}, ${content}, ${author}) RETURNING *`;
+      const newBlog = { title, discription, content, author };
+      return res.status(201).json(newBlog);
```

Comment:
instead of returning the same info back, we just redirect them to all blogs page